prow/cmd/deck/template/plugins.html

```diff
@@ -4,6 +4,7 @@
 <link rel="stylesheet" href="/static/dialog-polyfill.css">
 <script type="text/javascript" src="/static/plugin_help_bundle.min.js"></script>
 <script type="text/javascript" src="plugin-help.js?var=allHelp"></script>
+<script type="text/javascript" src="https://cdn.jsdelivr.net/gh/google/code-prettify@master/loader/run_prettify.js?autorun=false&skin=desert"></script>
```

Comment:
I'm not wildly in love with this being pulled from a random CDN and then miraculously appearing as a global, but it does appear to be the intended usage and basically the norm for syntax highlighters.

Reply:
As a more stable alternative, we can [server this statically](https://github.com/google/code-prettify/blob/master/docs/getting_started.md#serving-your-own-js--css) in `deck` if that is the preferred approach.

Reply:
@Katharine 
I added [`code-prettify` via npm](https://github.com/kubernetes/test-infra/pull/13507/commits/07677346e79fe16687bd341a48a186fa50e89768#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3) and am now [statically hosting the css](https://github.com/kubernetes/test-infra/pull/13507/commits/07677346e79fe16687bd341a48a186fa50e89768#diff-c2fc5d5c9336bcb9f695e01b191f4929R1).